src/harness/fourslashImpl.ts

```diff
@@ -4510,6 +4510,52 @@ export class TestState {
 
         this.verifyCurrentFileContent(newFileContent);
     }
+
+    public baselineMapCode(
+        ranges: Range[],

```

Comment:
Please take Range[][] here instead

Add tests where we are passing [[Range0, range1], [range2]] 
Add tests when active file name is "index.js"